CONTRIBUTING.md

```diff
@@ -20,3 +20,7 @@ specifier is not provided.
 Prefer `external` to `public` if you know a function will only be called externally. If there is a possibility
 of a function being called internally as well as externally, the function should be `public`. `external` functions can be more
 efficient when a function parameter is a large array of data.
+
+## Development Environment
+
+For easy development, we recommend using [Visual Studio Code](https://code.visualstudio.com/) with the [Hardhat extension](https://hardhat.org/hardhat-vscode/docs/overview). This will provide you with syntax highlighting, code completion, debugging capabilities, and an easy-to-use test explorer.
```

Comment:
Just a note that I've had mixed experiences with the VSCode Solidity extensions. 

IIRC the official Solidity extension worked well for most things but navigating through the code ("go to implementation" kind of thing) and the Hardhat one fixed that but broke other things. Maybe it's because we previously didn't have a `remappings.txt` file, not sure.

If you are getting a clean dev xp with that setup, let's update this doc with it though.